style.css

```diff
@@ -356,59 +367,1108 @@ span {
+  }
+
+  .aboutme-cards {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    list-style: none;
+    gap: 0;
+    padding: 0 0 0 0;
+  }
+
+  .aboutme-img-d {
+    background-image: url('./images/image_dots_with_teal.png');
+    background-repeat: no-repeat;
+    background-size: 65% 80%;
+    position: absolute;
+    left: 35%;
+    top: 21.5%;
+  }
+
+  .aboutme {
+    display: flex;
+    flex-direction: row-reverse;
+    padding: 4%;
+    justify-content: space-around;
+    gap: 1rem;
   }
 
   .aboutme-techs,
   .aboutme-framew,
-  .aboutme-skills {
+  .aboutme-skills,
+  .aboutme-img-d {
+    margin: 0 0 0 0;
     width: 100%;
-    margin-right: 1rem;
+    padding-top: 1rem;
+    position: relative;
+  }
+
+  .logo-corner1,
+  .logo-corner {
+    height: 90px;
+    left: 47.5%;
+    background-repeat: no-repeat;
+    background-size: contain;
+  }
+
+  .logo-corner2 {
+    left: 45%;
+  }
+
+  .techs,
+  .framew,
+  .skills {
+    font-size: 14px;
+    margin: 0 0.2rem 0 0;
+  }
+
+  .aboutme-title {
+    font-size: 90px;
+  }
+
+  .aboutme-btn {
+    width: 254px;
+  }
+
+  .contact-form-d {
+    position: relative;
+    display: flex;
+    justify-content: space-evenly;
+    height: 950px;
+    background: #f9f9fa;
+  }
+
+  .contact-title-d {
     display: flex;
     flex-direction: column;
     align-items: center;
+    width: 100%;
   }
 
-  .aboutme-cards {
-    padding-right: 1rem;
+  .contact-card {
+    width: 70%;
+    padding: 0;
+    margin-top: 20%;
   }
 
-  .aboutme-p {
+  .contact {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    width: 100%;
+    align-items: center;
+    gap: 0;
+  }
+
+  .contact-logo {
+    display: none;
+  }
+
+  .contact-logo1-d {
+    display: block;
+  }
+
+  .contact-logo2-d {
+    display: block;
+  }
+
+  .contact1-img {
+    max-width: 100%;
+    height: 26%;
+    position: absolute;
+    bottom: 0;
+    right: calc(80% - 1rem);
+    top: 0;
+  }
+
+  .contact-img {
+    max-width: 100%;
+    height: 26%;
+    position: absolute;
+    bottom: 0;
+    left: calc(72.5% - 1rem);
+    top: 74%;
+  }
+
+  .contact input[type="text"] {
+    width: 80%;
+    height: 52px;
+    background: #fff;
+    border-radius: 4px;
+  }
+
+  .contact textarea[name="your message"] {
+    width: 170%;
+  }
+
+  .input-d1,
+  .input-d2,
+  .input-d3 {
+    display: flex;
+    flex-direction: column;
+    gap: 0.5rem;
+    width: 100%;
+  }
+
+  .input-d1 {
+    margin-left: 2rem;
+  }
+
+  .input-d2 {
+    margin-right: 2rem;
+  }
+
+  .input-d3 {
+    margin-left: 2rem;
+  }
+
+  .form-btn {
+    width: 60%;
+    align-self: center;
+    margin-left: 53%;
+  }
+
+  .contact-title {
+    font-size: 90px;
+    margin: 1rem 0 0 0;
+  }
+
+  .contact-para {
+    width: 80%;
+    margin-bottom: 0 0 2rem 0;
+  }
+
+  .contact-footer {
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  .get {
+    margin: 4rem;
+  }
+
+  .contact-social {
+    margin: 5rem 10rem 0 0;
+  }
```

Comment:
- [ ] Please 🙏🏽 correct the button name it is `Get my CV`, align it to the center, and correct the margin between icons and their size to be similar to the Figma design.

| Figma design | Your design |
|---|---|
| ![image](https://github.com/hydaline-code/Portfolio_mobilefirst/assets/58642949/8ed419f2-02f4-4ad7-9b74-6d3d3981f221) | ![image](https://github.com/hydaline-code/Portfolio_mobilefirst/assets/58642949/ff9f226c-f143-4c0f-93e2-f26ab3566034) |